src/main/java/uk/gov/ons/ctp/response/action/printfile/LetterEntry.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.ons.ctp.response.action.printfile;
+
+import java.util.UUID;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.ons.ctp.response.action.representation.ActionTemplateDTO.Handler;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
+public class LetterEntry {
```

Comment:
Could we re-use PrintFileEntry rather than have a new class?

They look pretty much the same and I prefer the print file terminology

Reply:
The reason for a seperate one is to make the deletion easier. Also to make sure that the new changes doesn't interact with anything which is being used by the previous version. Which will still be used till the time we create the migration script.

Reply:
Ok that makes sense, I prefer the PrintFileEntry name as that what it is so maybe during the clean up we could rename it